scripts/custodian/Dockerfile

```diff
@@ -28,7 +28,9 @@ ENV DONOTDELETE_KEYS=$DONOTDELETE_KEYS
 ENV CUSTODIAN_YAML=aws.yaml
 
 ENV GOOGLE_APPLICATION_CREDENTIALS=google_credentials.json
-ENV GOOGLE_CLOUD_PROJECT=ei-container-platform-qa
+
+ARG GOOGLE_CLOUD_PROJECT
+ENV GOOGLE_CLOUD_PROJECT=GOOGLE_CLOUD_PROJECT
```

Comment:
I don't know why this looks weird to me... is the ENV supposed to be `GOOGLE_CLOUD_PROJECT=${GOOGLE_CLOUD_PROJECT}`?

Reply:
ah, was missing the $